data/json/mutations/mutations.json

```diff
@@ -3422,7 +3436,7 @@
     "visibility": 3,
     "ugliness": 1,
     "description": "You've gained a strong ability to leap from a standing position.",
-    "spells_learned": [ [ "spell_short_leap", 1 ] ],
+    "spells_learned": [ [ "spell_feline_leap", 1 ] ],
```

Comment:
Yes because as said it's a different version with a shorter cooldown and shorter range.

Reply:
But what you've done here is made both the `FELINE_LEAP` and `LEAPING_LEGS` traits have the exact same spell.   If I'm understanding you correctly your change to line 3425 needs to be reverted out because you are making a change to the Rabbit and Batrachian categories that you were not intending to make.

Reply:
Oops! Yeah change that!